lib/utils/config/definitions.js

```diff
@@ -239,15 +239,14 @@ define('audit-level', {
 
 define('auth-type', {
   default: 'legacy',
-  type: ['legacy', 'webauthn', 'sso', 'saml', 'oauth'],
+  type: ['legacy', 'web', 'sso', 'saml', 'oauth', 'webauthn'],
   // deprecation in description rather than field, because not every value
   // is deprecated
   description: `
-    NOTE: auth-type values "sso", "saml", and "oauth" will be removed in a future version.
-    
-    What authentication strategy to use with \`login\`.
+    NOTE: auth-type values "sso", "saml", "oauth", and "webauthn" will be
+    removed in a future version.
 
-    Pass \`webauthn\` to use a web-based login.
+    What authentication strategy to use with \`login\`.
```

Comment:
```suggestion
     Pass \`web\` to use a web-based login.
```

Reply:
I don't think removing the actual description is what we want here.